tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -316,10 +316,10 @@ void test_month_formatter() {
     assert(format(STR("{:%m %Om}"), month{1}) == STR("01 01"));
 
     // Out of bounds month
-    throw_helper(STR("{:%m}"), month{0});
+    assert(format(STR("{:%m}"), month{0}) == STR("00"));
     throw_helper(STR("{:%b}"), month{0});
     throw_helper(STR("{:%h}"), month{0});
-    throw_helper(STR("{::%B}"), month{0});
+    throw_helper(STR("{:%B}"), month{0});

```

Comment:
Was this intentional?

Reply:
Yes, I am having some weird problems where something is making me double type things on single keystrokes. Maybe it's vscode because it doesn't seem to happen very often on the browser.

Reply:
Wait so this _wasn't_ intentional?

Reply:
Removing the extra `:` was intentional.